dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -627,13 +627,7 @@ def get_airflow_extras():
     # END OF EXTRAS LIST UPDATED BY PRE COMMIT
 ]
 
-CHICKEN_EGG_PROVIDERS = " ".join(
-    [
-        "common.messaging",
-        "fab",
-        "openlineage",
-    ]
-)
+CHICKEN_EGG_PROVIDERS = " ".join(["common.messaging", "fab", "openlineage", "git"])
```

Comment:
`openlineage` can be removed, yeah?

Reply:
There is no easy answer. 

It really depends - it **Really** depends on some nuances there. There are cases where we have a crucial fix in the provider that will make us to **want** to use the "source" version of provider. We are just voting on 2.1.3rc1 of openlineage and the release notes are:

```
2.1.3
.....

Misc
~~~~

* ``Move ObjectStoragePath and attach to Task SDK (#48906)``
* ``Make '@task' import from airflow.sdk (#48896)``
```

The ObjectStoragePath move and import from `airflow.sdk` makes me think that we should also rather test thing and generate constraints in PyPI with the "latest" openlineage - that comes from sources. 

Do you think it's "save" to use 2.1.2 of openlineage that is currently released :) ?